guides/release-notes.md

```diff
@@ -7,6 +7,16 @@ We love hearing your feedback! You can help us improve the docs by reporting any
 ![Was this helpful, yes/no buttons](../media/feedback.png)
 
 ---
+### Updated API: PanelButton Element
+
+`Wix Editor Elements ($w)`
+
+We added following properties to the [`PanelButton`](https://www.wix.com/velo/reference/$w/panelbutton) element: 
++ [link](https://www.wix.com/velo/reference/$w/panelbutton/link): Sets or gets the element's link to an external site.
```

Comment:
Code font for link and target, so [`link`](https://www.wix.com/velo/reference/$w/panelbutton/link).